packages/vanilla-components/src/components/referral-list/referral-list.feature

```diff
@@ -0,0 +1,103 @@
+@owner:sam
+@author:sam
+Feature: Referral List
+
+  The Referral list component shows users a list of their referrals.
+
+  @motivating
+  Scenario: Referrer widget only shows referrals/rewards for associated program
+    Given I have made referrals in an attribution program "Program A" for user "A"
+    And I have made referrals in a different attribution program for "A"
+    And I have made referrals in the classic program for "A"
+    And I have manually made rewards for "A"
+    When I load the "Program A" referrer widget for "A"
+    Then I should only see the referrals associated with "Program A"
+
+
+  @ui
+  Scenario: "Referred by" users status label is "referred" until rewarded
+    Given I was referred to a program
+    When I load my program widget
+    Then the user who referred me in the referral list is shown
+    And the referral status for that user is not "referral pending"
+    But the referral status for that user is "referred"
+    And it is colored in green
+    When I earn a reward for being referred by that user
+    Then the referral status for that user matches the reward earned
+    And the status is colored in green
+
+
+  @ui
+  Scenario: Users I refer have the status "Reward Pending" until rewarded
+    Given I referred someone to a program
+    When I load my program widget
+    Then the user I referred in the referral list is shown
+    And the referral status for that user is "Reward Pending"
+    And the referral status is colored gray
+    When I earn a reward for referring that user
+    Then the referral status for that user matches the reward earned
+    And the status is colored in green
+
+
+  @ui
+  Scenario: Users I refer have the status "Reward Pending" until converted
+    Given I referred someone to a program
+    And the program does not reward the referrer
+    When I load my program widget
+    Then the user I referred in the referral list is shown
+    And the referral status for that user is "Reward Pending"
+    And the referral status is colored gray
+    When the referred user triggers the conversion
+    Then the referral status for that user says "Referred"
+    And the status is colored in green
+
+  @ui
+  Scenario Outline: Referral reward description
+    Given I have a referral
+    And the number of rewards for that referral is <numberOfRewards>
```

Comment:
This spec is not proper, there are more variables in the examples then used in the specs - some steps are clearly missing. I do believe this is a historical spec, but might be worth fixing up quickly

Reply:
decided to remove the extra columns (expiryText, expiryDate). i was trying to make sense of them and some were just wrong / didn't make sense and others redundant based on the rewardDescription column